plugins/aws/resoto_plugin_aws/aws_client.py

```diff
@@ -41,6 +41,9 @@ def __init__(
         self.role = role
         self.profile = profile
         self.region = region
+        self.AWS_ACCESS_KEY_ID = None
+        self.AWS_SECRET_ACCESS_KEY = None
+        self.AWS_SESSION_TOKEN = None
```

Comment:
I don't think this is a good idea. Is there a better way to achieve this without changing production code?

Reply:
@aquamatthias all of this is removed, credentials are now being used via the corresponding credentials file (or other mechanisms that I don't know about )